tvb/recon/cli/mrtrix.py

```diff
@@ -165,11 +167,34 @@ class tck2connectome(BaseMtrixCLI):
     """
 
     class Flags(BaseMtrixCLI.Flags):
-        assignment_radial_search = "-assignment_radial_search"
-        assignment_end_voxels = "-assignment_end_voxels"
+        out_assignments = "-out_assignments"
         scale_length = "-scale_length"
         stat_edge = "-stat_edge"
 
+    class Assignment(metaclass=abc.ABCMeta):
```

Comment:
but again, verbosity when intially writing the code is OK if the result is more readable / maintainable.  Since you're working on it, I let you choose approach you think is best.